### DISCUSSION TITLE
Engelsk version kører ustabilt....

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#8] [Bugs]
Vi får forskellige meldinger om langsom loadetid og sider i backenden, der ikke vil loade. Vi ved, at det er en fællesnævner, at det gælder det engelsksprogede arrangørdashboard, men ved ikke, om det er relevant. 

Jeg kan pt. bekræfte, at jeg selv oplever en lille smule længere loadetid på den engelske backend ift. den danske.

**Comment by antonlund18** [comment]
Hmm, det lyder meget mærkeligt - hvis noget, burde den engelske version faktisk loade hurtigere. Har du nogle eksempler på arrangører eller sider, som ikke gider loade / loader langsommere?

**Comment by MartinQwist** [comment]
F.eks. Emmas bruger på vores egen Kulturens Frivillige loader langsomt. Men det er markant bedre end i går, men altså stadig kun den engelske. 2-3 sekunder, så ikke noget kritisk. Bare lidt irriterende....

**Comment by antonlund18** [comment]
Jeg er stadig lidt i tvivl om hvilken side der er tale om. Eller er det alle sider i backenden på engelsk? Jeg synes ikke lige jeg kunne mærke en forskel, da jeg prøvede at ændre mit sprog til engelsk.
